aspnet/tutorials/nano-server.rst

```diff
@@ -1,8 +1,164 @@
-.. include:: /../common/stub-topic.txt
+.. _nano-server:
 
-|stub-icon| ASP.NET 5 on Nano Server
-====================================
+ASP.NET 5 on Nano Server
+========================
+
+By `Sourabh Shirhatti`
```

Comment:
Optional: update authors.txt and add a link for your name. Your github user works.